src/averageAges.js

```diff
@@ -55,10 +77,36 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
```

Comment:
The function name and parameter names could be more descriptive. Instead of 'mothers' and 'kids', consider using names that clearly indicate their role in the calculation.